### DISCUSSION TITLE
Is this protocol's traffic can be tracked by government?

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [juicity/juicity#116] [Q&A]
Is this protocol's traffic can be tracked by government? In my country every protocol is blocked.

**Comment by fbion** [comment]
u can try , it looks like udp quic 
where r u from?

**Comment by jiejianshiwaxinge** [comment]
> Is this protocol's traffic can be tracked by government? In my country every protocol is blocked.

Are you from Iran?
